src/guide/code-owners.md

```diff
@@ -53,58 +56,92 @@ This list is intended to make it easier to identify which email list to include
     * `libadt` – Compiler
     * `logging` – Runtime
     * `memory` – Runtime, GC
-    * `metaprogramming` – Runtime
-    * `oops` – Runtime
+    * `metaprogramming` – HotSpot
+    * `oops` – GC, Runtime
     * `opto` – Compiler
-    * `precompiled` – Runtime
+    * `precompiled` – HotSpot
     * `prims` – Runtime, Serviceability
     * `runtime` – Runtime
     * `services` – Runtime
-    * `shark` – Compiler
-    * `trace` – Runtime
-    * `utilities` – Runtime
+    * `utilities` – GC, Runtime
+  * `share/cds` – Runtime
 * `java.base`
-  * Core Libs should almost always be included but LangTools, HotSpot, Security and/or I18n may also be involved.
+  * Core Libs should almost always be included but Java Language, HotSpot, Security and/or I18n may also be involved.
   * `classes`
```

Comment:
Why isn't the `share` directory included here as in `hotspot`? Are these supposed to be full path names? It seems we should be consistent.

Reply:
The `share` directory was excluded in `java.base` because the other directories next to `share` (`linux`, `aix` etc) all have the same directory structure in them as `share`, and all those directories with the same names are owned by the same area. We could perhaps add a `*` to show this more clearly?